main/src/cgeo/geocaching/utils/EmojiUtils.java

```diff
@@ -416,6 +422,30 @@ public static BitmapDrawable getEmojiDrawable(final EmojiPaint paint, final int
         }
     }
 
+    /**
+     * get a drawable with given "wantedSize" as both width and height (either from cache or freshly built)
+     * @param wantedSize wanted size of drawable (width and height) in pixel
+     * @param emoji codepoint of the emoji to display
+     * @return drawable bitmap with emoji on it
+     */
+    @NonNull
+    public static BitmapDrawable getEmojiDrawable(final int wantedSize, final int emoji) {
+
+        EmojiPaint p;
+        synchronized (EMOJI_PAINT_CACHE_PER_SIZE) {
+            p = EMOJI_PAINT_CACHE_PER_SIZE.get(wantedSize);
+            if (p == null) {
+                final Resources res = CgeoApplication.getInstance().getApplicationContext().getResources();
+                final Pair<Integer, Integer> temp = new Pair<>(wantedSize, wantedSize);
+                final Pair<Integer, Integer> markerDimensions = new Pair<>((int) (temp.first * 1.2), (int) (temp.second * 1.2));
```

Comment:
you don't need the intermediate step of creating `temp`, you can use `wantedSize * 1.2` directly in the second line

Reply:
done